plugins/snapshots/erofs/plugin/plugin.go

```diff
@@ -1,3 +1,5 @@
+//go:build linux
```

Comment:
erofs snapshotter can be used on the Darwin host too..

Reply:
Thanks, I did not realize that! Ok, I think I have things so they should work fine in that setting still now.